components/RadioGroup/RadioButton.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import ButtonGroupLabel from '../shared/ButtonGroupLabel';
+import HiddenRadio from '../shared/HiddenRadio';
+import Icon from '../Icon';
+import uniqId from '../shared/uniqId';
+
```

Comment:
What you think about using a generator function in uniqId module?
We could add these 2 lines here:

```suggestion
const DEFAULT_ID_PREFIX = 'radio-button-';
const ID_GENERATOR = uniqId(DEFAULT_ID_PREFIX);
```

Reply:
Done